components/Alert.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from '../styles/components/alert.module.scss';
+import Button from './ui/Button';
+
+const Alert = ({ message, visible, onClose }) => (
+  <div
+    className={visible ? [styles.alertWrapper, styles.show].join(' ')
```

Comment:
This expression is too complex and should be refactored somehow. Imagine that in future you will need to have a few classes in case when alert is hidden. Also `styles.alertWrapper` is duplicated for both hidden and visible cases.

Maybe something like this would be better:
```js
const classList = [styles.alertWrapper];

if (visible) {
  classList.push(styles.show);
}
```

```jsx
 <div className={classList.join(' ')}>...</div>
```

Reply:
ok